segment_os.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+type osInfo struct {
+	props          *properties
+	env            environmentInfo
+}
+
+const (
+	//Macos the string to use for macOS
+	macOS Property = "macos"
+	//LinuxIcon the string to use for linux
+	Linux Property = "linux"
+	//WindowsIcon the icon to use for windows
+	Windows Property = "windows"
```

Comment:
this no longer references the `icon` notion, on propose?

Reply:
That's correct.  It defaults to the icon for the property, but I wanted to remove "icon" from the name so folks wouldn't think it could only be an icon.  Maybe that makes it more obvious that "win" would be a valid value, and not just an icon.

Reply:
Alright. Makes sense. I might have to update that everywhere to align (could break custom themes).

Reply:
are those in use somewhere already?